HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,571 @@
+            }
+        },
+        "ManagerAccount": {
+            "PropertyRequirements": {
+                "AccountTypes": {},
+                "Enabled": {},
+                "Links": {
+                    "PropertyRequirements": {
+                        "Role": {}
+                    }
+                },
+                "Password": {},
+                "PasswordChangeRequired": {},
+                "PasswordExpiration": {},
+                "RoleId": {},
+                "UserName": {}
+            }
+        },
+        "ManagerNetworkProtocol": {
+            "PropertyRequirements": {
+                "HostName": {},
+                "FQDN": {},
+                "HTTP": {
+                    "ReadRequirement": "IfImplemented",
+                    "PropertyRequirements": {
+                        "ProtocolEnabled": {},
+                        "Port": {}
+                    }
+                },
+                "HTTPS": {
+                    "PropertyRequirements": {
+                        "Certificates": {},
+                        "ProtocolEnabled": {},
+                        "Port": {}
+                    }
+                },
+                "NTP": {
+                    "ReadRequirement": "IfImplemented",
+                    "PropertyRequirements": {
+                        "NetworkSuppliedServers": {},
+                        "NTPServers": {},
+                        "Port": {},
+                        "ProtocolEnabled": {}
+                    }
+                },
+                "SSH": {
+                    "ReadRequirement": "IfImplemented",
+                    "PropertyRequirements": {
+                        "ProtocolEnabled": {},
+                        "Port": {}
+                    }
+                },
+                "SSDP": {
+                    "ReadRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "ProtocolEnabled": {},
+                        "Port": {},
+                        "NotifyMulticastIntervalSeconds": {},
+                        "NotifyIPv6Scope": {},
+                        "NotifyTTL": {}
+                    }
+                }
+            }
+        },
+        "OutboundConnection": {
+            "ReadRequirement": "Recommended",
+            "PropertyRequirements": {
+                "Authentication": {},
+                "Certificates": {},
+                "ClientCertificates": {},
+                "ConnectionEnabled": {},
+                "EndpointURI": {},
+                "Links": {
+                    "PropertyRequirements": {
+                        "Session": {}
+                    }
+                },
+                "PreUpgradeHTTPHeaders": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Roles": {},
+                "Status": {},
+                "WebSocketPingIntervalMinutes": {}
+            }
+        },
+        "RegisteredClient": {
```

Comment:
Same comments as OutboundConnection

Reply:
Scratch that; since registered clients are not mandatory, leave as Recommended. If we ever elevate them to be mandatory, then this needs to be IfPopulated.